### DISCUSSION TITLE
Can windows and iOS get along at all?

### DISCUSSION THREAD

**Discussion by heather-loll** [NativeScript/NativeScript#9963] [Q&A] [answered]
Hey! 

I'm totally new to mobile development and am coming here from the web development community. I program on windows and have an iPhone and have mostly been using the nativescript playground to mess around. Is there any way to move this workflow with using an iOS device with nativescript onto windows or are the platforms totally and completely incompatible? If so then how is it that the playground accomplishes this?

Thanks!

**Comment by rigor789** [comment, marked as answer]
Developing for ios is exclusive to macos, as all the tooling (xcode, simulator) is targeted at macos only. Playground specifically works around this (although with limitations) by replacing the javascript part of the app dynamically. Our playground successor is in private beta called Preview 2.0 that finally brings the latest NativeScript into the mix (sign up here if interested: https://nativescript.org/preview-private-beta) and brings much more flexibility to the table, specifically, you can use it against a local project you are working on in VSCode (or any editor). Limitations are still in place due to the way this works, however we do have plans to provide solutions for end-to-end ios development from non-macos systems (a mac will still be involved, but managed by us in the cloud) Nothing specific I can share about this yet, but do know we are thinking about the problem and are working towards a solution!
